gatsby-node.esm.js

```diff
@@ -378,6 +378,7 @@ exports.createPages = async ({actions, graphql}) => {
           name
           status
           a11yReviewed
+          importPath
```

Comment:
@camertron I am not super sure about the graphql schema updates. Feel free to edit it on the PR or suggest changes if they are not correct.